.github/workflows/ci-build.yml

```diff
@@ -1,33 +0,0 @@
-name: Build
```

Comment:
Why is this deleted? I thought we discussed at the Community Call...

Reply:
I understood that it is no longer needed.

Reply:
@duncdrum Could you comment as you set this up previously? I came away from the Community Call with a different understanding than Juri...